dotcom-rendering/src/model/article-schema.json

```diff
@@ -4229,7 +4229,7 @@
                     "type": "string"
                 },
                 "pillar": {
-                    "$ref": "#/definitions/LegacyPillar"
+                    "type": "string"
```

Comment:
But why?

Reply:
Basically on Frontend these are normally passed as pillars with the first letter capitalised, but for unknown reasons they're lowercase in the fronts response. This is a legacy type which we don't actually use anywhere in the codebase, and only exists to ensure our fixtures don't think TS errors. We should prioritise removing this type from frontend to solve this